ibis/backends/tests/test_array.py

```diff
@@ -473,19 +529,50 @@ def test_array_contains(backend, con):
     backend.assert_series_equal(result, expected, check_names=False)
 
 
+@pytest.mark.parametrize(
+    ("a", "expected_array"),
+    [
+        param(
+            [[1], [], [42, 42], []],
+            [-1, -1, 0, -1],
+            id="including-empty-array",
+            marks=[
+                pytest.mark.notyet(
+                    ["flink"],
+                    raises=Py4JJavaError,
+                    reason=(
+                        "SQL validation failed" "Flink does not seem to support ARRAY[]"
```

Comment:
Any particular reason why these two strings are separate? This will be automatically concatenated:

```
>>> "SQL validation failed" "Flink does not seem to support ARRAY[]"
'SQL validation failedFlink does not seem to support ARRAY[]'
```

which I doubt is what you intended

Reply:
Yes, they were initially in different lines and then placed in the same line by the linter. Merged them.